core/trino-main/src/main/java/io/trino/operator/aggregation/listagg/ListaggAggregationFunction.java

```diff
@@ -77,7 +77,6 @@ private ListaggAggregationFunction()
                                 .argumentType(BOOLEAN)
                                 .build())
                         .nullable()
-                        .argumentNullability(true, false, false, false, false)
```

Comment:
> "Fix null parameter declaration in listagg"

Add an explanation to the commit message. What's the issue? It's not obvious from this change